<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget</class>
 <widget class="QWidget" name="widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>163</width>
    <height>217</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="btnDebug">
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnInfo">
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnWarning">
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnCritical">
     <property name="text">
      <string>Critical</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="btnFatal">
     <property name="text">
      <string>Fatal</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
